springcloud/src/test/java/org/axonframework/extensions/springcloud/commandhandling/SpringCloudCommandRouterTest.java

```diff
@@ -168,6 +168,22 @@ public void testUpdateMembershipUpdatesLocalServiceInstance() {
                                                                                                  .contains(
                                                                                                          SERVICE_INSTANCE_ID))));
     }
+    
+    @Test
+    public void testUpdateMembershipUpdatesLocalServiceInstanceWhenMetadataNull() {
+        //Mocking null local service instance metadata for this test only
+        when(localServiceInstance.getMetadata()).thenReturn(null);
+        
+        CommandMessageFilter commandNameFilter = new CommandNameFilter(String.class.getName());
+        testSubject.updateMembership(LOAD_FACTOR, commandNameFilter);
+
+        verify(consistentHashChangeListener, never()).onConsistentHashChanged(argThat(item -> item.getMembers()
```

Comment:
I'd replace this for `verifyZeroInteractions(consistentHashChangeListener);`.

Reply:
Done.